src/css/base.scss

```diff
@@ -30,6 +29,55 @@ header {
 
 main {
   font-family: 'Exo 2', sans-serif;
+
+  button {
+    font-family: 'Exo 2', sans-serif;
+    height: 30px;
+    width: 200px;
+    background-color: $accent-font;
+    color: white;
+    border: none;
+    cursor: pointer;
+    &:hover {
+      box-shadow: 0 0 10px white;;
+      transition: 0.4s;
+    }
+  }
+
+  #tabs_customer_rooms {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    align-items: center;
+  }
+
+  #tabs_customer_main {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    align-items: center;
+  }
```

Comment:
These css properties could be dried up into a mixin
```
    display: flex;
    flex-direction: column;
    text-align: center;
    align-items: center;
```

It would look something like this:
```
@mixin flex {
    display: flex;
    flex-direction: column;
    text-align: center;
    align-items: center;
}
```